src/console_reporter.cc

```diff
@@ -150,8 +150,15 @@ void ConsoleReporter::PrintRunData(const Run& result) {
   for (auto& c : result.counters) {
     const std::size_t cNameLen = std::max(std::string::size_type(10),
                                           c.first.length());
-    const double one_k =
-        double(c.second.thousand.first) / double(c.second.thousand.second);
+    const double one_k = [](benchmark::Counter::One_K k) -> double {
```

Comment:
why not give the enums the values '1000' and '1024' and pass them through (with any necessary cast) to the HumanReadableNumber call?

Reply:
There is no _real_ reason not to do it.
I didn't do that because `0` and `1` (which is most likely the current values of these values)
only need one bit, while `1024` needs 11 bits. So _if_ we wanted to pack the size of `Counter`,
it would _likely_ be simpler with smaller enum values.
I don't think we care about that here, so let's avoid this premature optimization :)